common/pixman-region.c

```diff
@@ -2624,166 +1631,3 @@ PREFIX (_extents) (region_type_t *region)
     return (&region->extents);
 }
 
-/*
- * Clip a list of scanlines to a region.  The caller has allocated the
- * space.  FSorted is non-zero if the scanline origins are in ascending order.
- *
- * returns the number of new, clipped scanlines.
- */
-
-PIXMAN_EXPORT pixman_bool_t
-PREFIX (_selfcheck) (region_type_t *reg)
-{
```

Comment:
This function is used if building with `--enable-devel-logging` or `--eanable-devel-all` (and without `--enable-pixman`). Building with these options now gives me a link error.

Reply:
I thought I did a full tree search for all function before removing them. Must have been my recent sloppiness. Will add back.

Reply:
Looks like I already addressed this.